Spec/Auth.swift

```diff
@@ -1999,6 +1999,154 @@ class Auth : QuickSpec {
+                    
+                    let authOptions = ARTAuthOptions()
+                    authOptions.queryTime = true
+                    
+                    waitUntil(timeout: testTimeout) { done in
+                        rest.auth.createTokenRequest(nil, options: authOptions, callback: { tokenRequest, error in
+                            expect(error).to(beNil())
+                            guard let tokenRequest = tokenRequest else {
+                                XCTFail("TokenRequest is nil"); return
+                            }
+                            expect(tokenRequest.timestamp).toNot(beNil())
+                            expect(serverTimeRequestWasMade).to(beTrue())
+                            expect(rest.auth.timeOffset).toNot(beNil())
+                            done()
+                        })
+                    }
+                    
+                    var serverTimeRequestWasMade2 = false
+                    let block2: @convention(block) (AspectInfo) -> Void = { _ in
+                        serverTimeRequestWasMade2 = true
+                    }
+                    
+                    let hook2 = ARTRest.aspect_hookSelector(rest)
+                    let _ = try? hook2(#selector(ARTRest.time(_:)), withOptions: .PositionAfter, usingBlock:  unsafeBitCast(block2, ARTRest.self))
+                    
+                    waitUntil(timeout: testTimeout) { done in
+                        rest.auth.createTokenRequest(nil, options: authOptions, callback: { tokenRequest, error in
+                            expect(error).to(beNil())
+                            guard let tokenRequest = tokenRequest else {
+                                XCTFail("TokenRequest is nil"); return
+                            }
+                            expect(tokenRequest.timestamp).toNot(beNil())
+                            expect(serverTimeRequestWasMade2).to(beFalse())
+                            done()
+                        })
+                    }
+                }
+                
+                it("should be possible by lib Client to discard the cached local clock offset") {
+                    let rest = ARTRest(options: AblyTests.commonAppSetup())
+                    
+                    let authOptions = ARTAuthOptions()
+                    authOptions.queryTime = true
+                    
+                    waitUntil(timeout: testTimeout) { done in
+                        rest.auth.createTokenRequest(nil, options: authOptions, callback: { tokenRequest, error in
+                            expect(error).to(beNil())
+                            guard let tokenRequest = tokenRequest else {
+                                XCTFail("tokenRequest is nil"); return
+                            }
+                            expect(rest.auth.timeOffset).toNot(beNil())
+                            rest.auth.discardTimeOffset()
```

Comment:
This is not needed.  It's saying the it's possible that the "Client" should be able to discard the local clock offset, not the user.  Anyway, leave now, but this spec item was simply a statement to make it clear that the client library developer when writing the client could under certain circumstances choose to discard the offset internally.